src/rfkill/rfkill.c

```diff
@@ -171,14 +171,17 @@ static int load_state(Context *c, const struct rfkill_event *event) {
                 .op = RFKILL_OP_CHANGE,
                 .soft = b,
         };
+        assert_cc(offsetof(struct rfkill_event, op) < RFKILL_EVENT_SIZE_V1);
+        assert_cc(offsetof(struct rfkill_event, soft) < RFKILL_EVENT_SIZE_V1);
 
         ssize_t l = write(c->rfkill_fd, &we, sizeof we);
```

Comment:
use RFKILL_EVENT_SIZE_V1 as 3rd param here?

Reply:
I vacillated a bit on this, but in the end I think it's nice to use the full size because then we get information what the kernel supports in the logs. The kernel is actually fine (both old and new versions) with whatever size.